iib/web/models.py

```diff
@@ -212,10 +210,10 @@ class RequestArchitecture(db.Model):
 
     # A primary key is required by SQLAlchemy when using declaritive style tables, so a composite
     # primary key is used on the two required columns
-    request_id = db.Column(
+    request_id: Mapped[int] = db.mapped_column(
         db.Integer, db.ForeignKey('request.id'), autoincrement=False, index=True, primary_key=True
```

Comment:
Is this db.Integer parameter needed with Mapped[int]?

Reply:
thanks for catching this. I have updated such instances.